exercises/isogram/README.md

```diff
@@ -0,0 +1,72 @@
+# Isogram
+
+Determine if a word or phrase is an isogram.
+
+An isogram (also known as a "nonpattern word") is a word or phrase without a repeating letter.
+
+Examples of isograms:
+
+lumberjacks
+background
+downstream
+The word isograms, however, is not an isogram, because the s repeats.
+
+## Getting Started
+
+For installation and learning resources, refer to the
+[exercism help page](http://exercism.io/languages/haskell).
+
+## Running the tests
+
+To run the test suite, execute the following command:
+
+```bash
+stack test
+```
+
+#### If you get an error message like this...
+
+```
+No .cabal file found in directory
+```
+
+You are probably running an old stack version and need
+to upgrade it.
+
+#### Otherwise, if you get an error message like this...
+
+```
+No compiler found, expected minor version match with...
+Try running "stack setup" to install the correct GHC...
+```
+
+Just do as it says and it will download and install
+the correct compiler version:
+
+```bash
+stack setup
+```
+
+## Running *GHCi*
+
+If you want to play with your solution in GHCi, just run the command:
+
+```bash
+stack ghci
+```
+
+## Feedback, Issues, Pull Requests
+
+The [exercism/haskell](https://github.com/exercism/haskell) repository on
+GitHub is the home for all of the Haskell exercises.
+
+If you have feedback about an exercise, or want to help implementing a new
+one, head over there and create an issue.  We'll do our best to help you!
+
+## Source
+
+see more  at [wikipedia page of isograms]("https://en.wikipedia.org/wiki/Isogram")
```

Comment:
as you can see in https://github.com/Average-user/haskell/blob/d1556eadc748688089e3261700ab9678f2caf245/exercises/isogram/README.md, the quotes mean it is not a link.

I'm going to regenerate this README anyway so it doesn't matter whether you fix this, but don't put quotes in the brackets

Reply:
ok, so the next time i make PR like this, do i even have to put the README.md?

Reply:
I guess not! I will just do it. However, it would be good if you just put a README.md that just says "please regenerate this for me" so that I don't forget to!

Reply:
ok, also, dont merge yet, i want to implement something